modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Tabs/BasicInfo/BasicInfoTab.tsx

```diff
@@ -216,33 +214,32 @@ export function BasicInfoTab({
 				/>
 			</ContainerWrapper>
 
-			{Liferay.FeatureFlags['LPS-135430'] && !isDefaultStorageType && (
-				<ContainerWrapper
-					collapsable
-					defaultExpanded
-					displayTitle={Liferay.Language.get('external-data-source')}
-					displayType="unstyled"
-					title={Liferay.Language.get('external-data-source')}
-				>
-					<Input
-						className={classNames({
-							'lfr-objects__edit-object-field-model-builder-panel':
-								modelBuilder,
-						})}
-						label={Liferay.Language.get('external-reference-code')}
-						name="externalReferenceCode"
-						onBlur={(event) => {
-							event.stopPropagation();
-
-							if (onSubmit) {
-								onSubmit();
-							}
-						}}
-						onChange={handleChange}
-						value={values.externalReferenceCode}
-					/>
-				</ContainerWrapper>
-			)}
+			<ContainerWrapper
+				collapsable
+				defaultExpanded
+				displayTitle={Liferay.Language.get('external-data-source')}
+				displayType="unstyled"
+				title={Liferay.Language.get('external-data-source')}
+			>
+				<Input
+					className={classNames({
+						'lfr-objects__edit-object-field-model-builder-panel':
+							modelBuilder,
+					})}
+					disabled={values.system || isApproved}
```

Comment:
Hey, I talked to Ana from the product team and she said this should be editable even if it is published. So, the disable condition should **not** have the `isApproved` case. Only system fields should be disabled. Can you clarify why have you added the `isApproved`? Maybe you know something we don't.

Reply:
I thought it was supposed to work like the field name that gets disabled after publishing. Totally my bad for not clearing that up earlier.